extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -175,17 +180,19 @@ export class AmpAnalytics extends AMP.BaseElement {
             return;
           }
 
+          const instrumentation_ = instrumentationServiceForDoc(
```

Comment:
Resolve the instrumentation service once in the `buildCallback` and use private var reference.